### ISSUE TITLE
Embed groovy-xml.jar by default

### ISSUE THREAD

**Issue by hilsonp** [outofcoffee/imposter#399]
You embed groovy-json and this is really useful for REST related groovy scripts.
Could you also embed groovy-xml that is de-facto needed for SOAP related groovy scripts as soon as we need to put some logic on the body part ? Especially since in SOAP, most of the things are part of the BODY.

Thank you.

**Comment by cmorasca** [comment]
We had the same need and we worked around creating a docker compose file to copy the library and create a new image. 

You can download the image from here:
https://mvnrepository.com/artifact/org.codehaus.groovy/groovy-xml

Create a Dockerfile with the following content:

```
FROM outofcoffee/imposter-all
COPY ./groovy-xml-3.0.9.jar /opt/imposter/lib
```

after that, create a docker compose file (docker-compose.yml) with the following content:

```
version: '3.9'
services:
  imposter:
    build:
      context: .
      dockerfile: Dockerfile
    container_name: imposter
    ports:
      - "8080:8080"
    volumes:
      - /YOUR/LOCAL/PATH:/opt/imposter/config
    tty: true
    stdin_open: true
```

to start the container you just type **docker compose up**. If you want to run it in daemon mode, type **docker compose up -d**

please note that this is just an example, and could be done in different ways.

Cheers!

**Comment by outofcoffee** [comment]
This is a sensible addition. It brings us closer to parity to the old `groovy-all` dependency, which included this (and the JSON dependency we already bundle).

Released in [v3.21.0](https://github.com/outofcoffee/imposter/releases).
